.flake8

```diff
@@ -1,21 +1,18 @@
 [flake8]
 min_python_version = 3.6.0
-ignore =
-    # E501: Line too long
+max-line-length = 88
+inline-quotes = double
+extend-ignore =
+    # E501: Line too long (FIXME: long string constants)
     E501,
-    # W503: Line break occurred before a binary operator
-    W503
+    # E203: Whitespace before ':' (pycqa/pycodestyle#373)
+    E203,
```

Comment:
any reason why we are adding this ignore now?

Reply:
It's part of the standard black config. We haven't triggered it, but black calls for it as it's a known conflict.

Reply:
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8